articles/organizations/index.md

```diff
@@ -0,0 +1,122 @@
+
+##### Roles
+
+Members of organizations can be [assigned roles](/organizations/configure-membership#add-roles-to-members), which are added to the member’s Access Token. You can use these roles to define access control for your application. For example, if you built a dashboard for your users using our API and SDKs, you could assign an administrator role to certain members and allow them to self-manage their organizations through your dashboard.
+
+::: warning
+If you build a custom dashboard for your product, note that API use is subject to Auth0 Management API rate limits.
+:::
+
+## Limitations
+
+Organizations currently has the following limitations:
+
+* Supported in only new Universal Login (cannot be used with classic Universal Login or Lock.js)
+* Does not support:
+  * custom domains per organization (For example, using the sample scenario, if Rocky Mountain High Adventures and Granite Outpost Rafting and Ziplining could both use `login.travel0.com` as login domains, then Organizations would be useful. Alternatively, if Rocky Mountain High Adventures wanted to use `login.rockymountain.com` and Granite Outpost wanted to use `login.graniteoutpost.com`, then we would need to use multiple Auth0 tenants instead.)
+  * the following grants and protocols: Client Credentials, SAML (Auth0 as IdP, WS-Fed (Auth0 as IdP)
+  * integration with the Delegated Administration extension
+  * integration with the Authorization Extension
+
+### Entity limits
+
+* Each organization may have a maximum of 10 metadata properties.
+* Each organization may have a maximum of 10 enabled connections.
+* Each organization member may have a maximum of 50 roles.
+* Up to 1000 organizations can be displayed using the Auth0 Dashboard or Management API, though more may exist.
+
+## Upcoming changes
+
+Organizations is currently in beta, and based on customer feedback, we have already identified several areas that will change:
+
+* **Organization connections on the Login prompt**: Currently, connections defined at the application-level and connections defined at the organization-level are merged and displayed on the organization Login prompt. Eventually, only the connections defined at the organization level will be displayed.
+* **Application URI parameters**: We plan to allow for parameterized application URIs (for example, callback URLs, login/logout URLs) that enable developers to allow-list URIs that contain organizations’ names in their subdomains.
+* **User invitation redirect URLs**: To allow custom redirection after an end-user has accepted a membership invitation, we plan to allow organizations to be assigned their own redirect URLs.
```

Comment:
* **User invitation redirect URLs**: We plan to modify the user invitation flow to allow more flexible redirection after an end-user has accepted a membership invitation.